srml/staking/src/lib.rs

```diff
@@ -553,6 +553,19 @@ decl_module! {
 		/// account that controls it.
 		///
 		/// The dispatch origin for this call must be _Signed_ by the stash account.
+		///
+		/// # <weight> FLAG
```

Comment:
Earlier you had FLAG as a bullet point. Should be a consistent formatting (especially if you have a script to find them)

Reply:
I think we'll coalesce into using this formatting for FLAGs since it will be easier to human parse. We don't have the script yet afaik.

Reply:
I think we'll coalesce into using this formatting for flags since it's easier to human parse and we haven't written the script yet.